env_vars/vagrant.yml

```diff
@@ -5,6 +5,13 @@ setup_git_repo: yes
 git_branch: master
 
 
+# Project layout
+
+# Uncomment and change this if you have specific requirements when
+# running the app on the Vagrant VM.
+# requirements_relative_file: "./requirements_local.txt"
```

Comment:
@StuartMacKay same issue here - shouldn't this be uncommented?

Reply:
I added the commented out variables to development.yml and vagrant.yml so that you'd have the option to override the default in base.yml for local development.

To run the playbook I have a JSON file with all the variables I need and pass it on the command line using --extra-vars="@<my variables>.json". That neatly solves the problem with Ansible vast and complicated precedence rules. 

As a result I am not sure that having extra definitions that can be selectively uncommented make any sense. I'm happy with any change you want here.